checkstyle/suppressions.xml

```diff
@@ -224,7 +228,7 @@
               files="SmokeTestDriver.java"/>
 
     <suppress checks="NPathComplexity"
-              files="EosTestDriver|KStreamKStreamJoinTest.java|RelationalSmokeTest.java|SmokeTestDriver.java|KStreamKStreamLeftJoinTest.java|KTableKTableForeignKeyJoinIntegrationTest.java"/>
+              files="EosTestDriver|KStreamKStreamJoinTest.java|RelationalSmokeTest.java|SmokeTestDriver.java|KStreamKStreamLeftJoinTest.java|KTableKTableForeignKeyJoinIntegrationTest.java|StoreChangelogReaderTest.java"/>
```

Comment:
All those additional exception indicate that we might need to do even more refactoring -- refactoring should reduce code complexity... (did we try to remove some exception to see what we gained? -- maybe it's just shifted?)

Reply:
Yes some of those are added temporarily during the development, and could be removed now, will do that.